G/GAP_pkg/GAP_pkg_normalizinterface/build_tarballs.jl

```diff
@@ -0,0 +1,46 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+include("../common.jl")
+
+gap_version = v"400.1191.001"
+gap_lib_version = v"400.1191.000"
+name = "NormalizInterface"
+upstream_version = v"1.3.0" # when you increment this, reset offset to v"0.0.0"
+offset = v"0.0.0" # increment this when rebuilding with unchanged upstream_version, e.g. gap_version changes
+version = offset_version(upstream_version, offset)
+
+# Collection of sources required to build libsingular-julia
+sources = [
+    ArchiveSource("https://github.com/gap-packages/$(name)/releases/download/v$(upstream_version)/$(name)-$(upstream_version).tar.bz2",
+                  "5fd7a1e13bbd48cd119456745f00f8208cd09109c0f6614f0165e1965b92afdd"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd NormalizInterface*
+
+atomic_patch -p1 ${WORKSPACE}/srcdir/patches/NormalizInterface.patch
```

Comment:
Wait, I'm incredibly confused: the only source is `NormalizInterface` tarball, where is this patch coming from?  And how are the other two files in `G/GAP_pkg/bundled` involved?